src/main/java/com/conveyal/r5/analyst/cluster/AnalystWorker.java

```diff
@@ -449,8 +465,8 @@ public void sleepSeconds (int seconds) {
                 // This is a single task within a regional analysis with many origins.
                 if (request.makeStaticSite) {
                     // This is actually a time grid, because we're generating a bunch of those for a static site.
-                    OutputStream s3stream = MultipointDataStore.getOutputStream(request, request.taskId + "_times.dat", "application/octet-stream");
-                    s3stream.write(oneOriginResult.timeGrid.writeGrid());
+                    LittleEndianDataOutputStream s3stream = MultipointDataStore.getOutputStream(request, request.taskId + "_times.dat", "application/octet-stream");
```

Comment:
I guess MultipointDataStore is always returning a LittleEndianDataOutputStream because all our regional results stored o S3 are always gzipped arrays of little endian ints. I would more expect it to return a general OutputStream, and for TimeGrid to be able to write to any old stream (by wrapping it with its own LittleEndianDataOutputStream internally). But I guess it already worked the way I'm seeing here and you were just maintaining the existing functionality.